packages/fluentui/react-builder/src/components/NarrationComputer.tsx

```diff
@@ -0,0 +1,268 @@
+/*
+ TODO:
+* Add the missing and not so obvious attributes (e.g. "aria-haspopup" or "aria-expanded") for the already defined roles (e.g. "menuitem" or "checkbox") according to the specification.
+* With JAWS, in the case of the element with the "listbox" role, differentiate between having and not having the aria-multiselectable="true" attribute. If this attribute is present, then aria-selected="false" on the child elements with the role "option" behave differently than if it is not present. Specifically, if aria-multiselectable="true" is present, the aria-selected="false" causes the narration of "not selected", but if present, having the aria-selected attribute makes no difference to the narration. 
+ * Should we also consider the "disabled" state?
+ */
+import SRNC from './SRNC-Definitions';
+import './SRNC-Rules-Win_JAWS';
+
+export default class NarrationComputer {
+  private computedParts: Record<string, string> = {
+    value: '',
+    name: '',
+    description: '',
+    type: '',
+    state: '',
+    position: '',
+    usage: '',
+  };
+
+  // Computes and returns the complete screen reader narration for the given element and platform.
+  async computeNarration(element: Element, platform: string): Promise<string> {
+    let definitionName = this.getDefinitionName(element, platform, 'stateRules');
+
+    // Retrieve the computed accessible node
+    const node = await (window as any).getComputedAccessibleNode(element);
```

Comment:
```suggestion
    const node = await (window as Window).getComputedAccessibleNode(element);
```
Maybe this is better type?

Reply:
The `getComputedAccessibleNode()` method is an experimental feature. When i change the type to `Window`, I get the following error: `Property 'getComputedAccessibleNode' does not exist on type 'Window'.`.

Reply:
@miroslavstastny or @layershifter Can we do some global module augmentation for Window here or just go with `any` anyway?

Reply:
I don't see any definitions for `getComputedAccessibleNode()` in [lib.dom.d.ts](https://github.com/microsoft/TypeScript/blob/master/lib/lib.dom.d.ts), so any is a single thing :(